settings.gradle.kts

```diff
@@ -10,11 +10,11 @@ include(
   "backend:api",
   "backend:impl",
   "frontend:base",
-  "frontend:actions",
-  "frontend:file",
   "frontend:resourcebundles",
+  "frontend:file",
```

Comment:
Just to keep topo-ish order

Reply:
wow, actions module was a bit high :weed: